docs/database/postgresql_scaffolding.sql

```diff
@@ -0,0 +1,39 @@
+-- create database
```

Comment:
@fmichonneau I thought I had lost this file, but turns out it was simply committed.